Pipfile

```diff
@@ -17,3 +17,6 @@ pytest = "*"
 pytest-timeout = "*"
 pytest-cov = "*"
 coala-bears = "*"
+
+[requires]
+python_version = "3.8"
```

Comment:
@fridex This repo didn't have `[requires]`, but I added, WDYT?

Reply:
Nice catch. Yes, It is good to have it to make sure the right python version is running. :+1: